src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {
+  useState,
+  ReactElement,
+  FC,
+  SyntheticEvent,
+  Children,
+  cloneElement,
+} from 'react';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: any | null;
+  defaultExpandedPanel?: number | null;
+}
+
+const Accordion: FC<AccordionProps> = ({
+  children,
+  defaultExpandedPanel,
+  ...rest
```

Comment:
Minor, naming thing: The word "rest" is like writing a function that takes "arg" - no semantic benefit. That's why I prefer `passedProps` (unless you're doing something other than passing the extra props down to another node).